iothub/service/samples/readme.md

```diff
@@ -1,14 +1,85 @@
-Service samples were moved to [Azure-Samples/azure-iot-samples-csharp][samples-repo] repository
+## Service samples for Azure IoT Service SDK for C#
 
-### [Service samples][service-samples]
-* [ADMSample][adm-sample]
-* [JobsSample][jobs-sample]
-* [RegistryManagerSample][reg-man-sample]
-* [ServiceClientSample][service-client-sample]
+Here, you'll find simple samples showing how to use the various features of the Microsoft Azure IoT Hub service from a back-end application running C# code.
+
+### List of Samples
+
+* [ADM Sample][adm-sample]
+* [Jobs Sample][jobs-sample]
+* [Registry ManagerSample][reg-man-sample]
+* [Service Client Sample][service-client-sample]
+* [Plug and play Samples ][pnp-service-sample]
```

Comment:
Add a link to other service SDK samples present here: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/master/iot-hub/Samples/service
You can leave out the `CleanUpDevicesSample`.

Reply:
sure